src/components/ContentCard/ContentCard.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import ContentCard from './ContentCard';
+import Tag from '../Tag/Tag';
+import Image from '../Image/Image';
+
+const meta = {
+  title: 'Components/ContentCard',
+  component: ContentCard,
+  parameters: {
+    layout: 'centered'
+  },
+  argTypes: {
+    variant: {
+      options: ['', 'border-light', 'border-ghost', 'ghost'],
+      control: { type: 'radio' }
+    }
+  },
+  tags: ['autodocs']
+} satisfies Meta<typeof ContentCard>;
+
+export default meta;
+type Story = StoryObj<typeof ContentCard>;
+
+export const Playground: Story = {
+  args: {
+    texts: {
+      cardTitle: 'Card title',
+      cardDescription:
+        'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis nec vestibulum augue, viverra aliquet nunc. Cras eget felis sodales, vestibulum neque ac, rhoncus ipsum.'
+    },
+    headingLevel: 'h2',
+    actionLink: false,
+    actionLinkText: 'Call-to-action',
+    icon: false,
+    iconName: 'demoCardIcon'
+  },
+  render: (args) => <ContentCard {...args} />
```

Comment:
Since we aren't doing anything special with the render, we can remove it. WE should only need the render method if we need more than `<Component {...args} />`